gov/governance/execute.gno

```diff
@@ -316,14 +317,14 @@ func callableMsg(pkgPath, function, params string) bool {
 			}
 			return true
 
-		case "GnsSetAvgBlockTimeInMs":
-			if len(param) != 1 {
-				panic(addDetailToError(
-					errInvalidFunctionParameters,
-					ufmt.Sprintf("execute.gno__callableMsg() || len(param) should be 1 but got %d, for param %s (pkgPath %s and function %s)", len(param), params, pkgPath, function),
-				))
-			}
-			return true
+		// case "GnsSetAvgBlockTimeInMs":
```

Comment:
What's up with comments?

Reply:
this was commented out because this part panicked with `GnsSetAvgBlockTimeInMs` is not exists error.

Reply:
please merge latest main branch.
it is removed function.

Reply:
merged https://github.com/gnoswap-labs/gnoswap/pull/358/commits/72d4d01dd0518270ddcee43ff05db5365111e8c3